app-backend/migrations/src_migrations/main/scala/54.scala

```diff
@@ -0,0 +1,15 @@
+import slick.driver.PostgresDriver.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M54 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(54)(List(
+    sqlu"""
+    CREATE TABLE organization_features (
+      organization UUID NOT NULL UNIQUE,
+      feature_flags UUID[],
```

Comment:
How do we know if a feature flag is enabled or disabled for an organization? Should that be tracked in this list - so a list of `{featureflag: UUID, enable: Boolean}` maybe?